command/agent/agent.go

```diff
@@ -279,6 +279,8 @@ func convertServerConfig(agentConfig *Config) (*nomad.Config, error) {
 		dur, err := time.ParseDuration(gcInterval)
 		if err != nil {
 			return nil, err
```

Comment:
Not your code, but would be nice to wrap this too
```suggestion
			return nil, fmt.Errorf("failed to parse job_gc_interval: %v", err)
```

The current error is a bit confusing when I set the duration to `qwers`:

```
==> Error starting agent: server config setup failed: time: invalid duration qwers
```